ludwig/utils/misc_utils.py

```diff
@@ -84,6 +84,10 @@ def resolve_pointers(dict1, dict2, dict2_name):
             if key_in_dict2 in dict2.keys():
                 value = dict2[key_in_dict2]
                 resolved_dict[key] = value
+            if ENCODER in dict2.keys():  # Temporary fix, this whole function should be removed during preproc refactor
+                if key_in_dict2 in dict2[ENCODER].keys():
+                    value = dict2[ENCODER][key_in_dict2]
+                    resolved_dict[key] = value
```

Comment:
It's a symptom of the `config_obj.to_dict()` returning a fully populated config. Since this PR passes in the return of `config_obj.to_dict()` to the preprocessing pipeline, there are now encoders in the feature configs, so this basically makes sure that the resolve pointers function works the same way with the additional params in the feature config. Will add a TODO to refactor this since it will go away once I refactor the preprocessing section with the Config Object.